be/src/http/action/stream_load.cpp

```diff
@@ -98,6 +98,8 @@ static TFileFormatType::type parse_format(const std::string& format_str,
         if (compress_type.empty()) {
             format_type = TFileFormatType::FORMAT_JSON;
         }
+    } else if (boost::iequals(format_str, "PARQUET")) {
```

Comment:
Better to use  `iequals` in `be/src/util/string_util.h`,  avoid using functions from boost